client/src/app/containers/Home/ActiveGamesTable.jsx

```diff
@@ -0,0 +1,118 @@
+import React from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { useHistory } from 'react-router-dom';
+import { useInjectReducer, useInjectSaga } from 'redux-injectors';
+import { selectCurrentUser } from '../../../auth/selectors';
+import { authSaga } from '../../../auth/saga';
+import {
+  sliceKey as authSliceKey,
+  reducer as authReducer,
+} from '../../../auth/slice';
+import { selectActiveGames } from './selectors';
+import { homeSaga } from './saga';
+import {
+  sliceKey as homeSliceKey,
+  reducer as homeReducer,
+  actions as homeActions,
+} from './slice';
+
+const ActiveGamesTable = () => {
+  useInjectReducer({ key: authSliceKey, reducer: authReducer });
+  useInjectSaga({ key: authSliceKey, saga: authSaga });
+  useInjectReducer({ key: homeSliceKey, reducer: homeReducer });
+  useInjectSaga({ key: homeSliceKey, saga: homeSaga });
+  const dispatch = useDispatch();
+  const currentUser = useSelector(selectCurrentUser);
+  const activeGames = useSelector(selectActiveGames(currentUser.id));
+
+  // event handlers
+  const onRefreshActiveGames = () => {
+    dispatch(homeActions.refreshActiveGames({ id: currentUser.id }));
+  };
+
+  let gameButtons = activeGames.map(activeGame => {
+    if (!activeGame || !activeGame.gameID) {
+      return;
+    }
+    const gID = activeGame.gameID;
+
+    let opponents = [];
+    for (const [pID, pName] of Object.entries(activeGame.players)) {
+      if (pID === currentUser.id) {
+        continue;
+      }
+      opponents.push(pName);
+    }
+
+    return (
+      <tr key={`gameRow ${gID}`}>
+        <td className='px-6 py-4 whitespace-no-wrap border-b border-gray-200'>
```

Comment:
I'd like to see this extracted to a component in `tailwind.css`. Perhaps we can do:

```css
.table-data {
  @apply px-6 py-4 whitespace-no-wrap border-b border-gray-200;
}

.table-data-sm {
  @apply text-sm leading-5 text-gray-500;
}
```
Then here we can use `className='table-data'.